examples/third_party/python/BUILD.python3.bazel

```diff
@@ -0,0 +1,43 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make(
+    name = "python3",
+    configure_env_vars = select({
+        "@platforms//os:macos": {"AR": ""},
+        "//conditions:default": {},
+    }),
+    configure_options = [
+        "--with-openssl=$EXT_BUILD_DEPS/openssl",
+        "--with-zlib=$EXT_BUILD_DEPS/zlibasdffdsaa",
```

Comment:
I wonder if it would have been better to pass random things to `args`. Is the fact that the build cannot find zlib the issue now? `args` are arguments to make which should be a way of introducing a new build variant that has no impact on the actual build itself.